kernel/kernel-api/src/test/scala/io/delta/kernel/internal/SnapshotManagerSuite.scala

```diff
@@ -135,6 +136,14 @@ class SnapshotManagerSuite extends AnyFunSuite with MockFileSystemClientUtils {
     topLevelFileTypes.foreach { topLevelFileType =>
       val v2Checkpoints =
         v2CheckpointFileStatuses(v2CheckpointSpec, topLevelFileType)
+
+      val v2CheckpointFileName = if (v2CheckpointSpec.nonEmpty) {
```

Comment:
isn't `v2Checkpoints` a list of v2 checkpoint files statuses? are we able to use that? do we need to re-make a new name?

Reply:
Yeah, I think each sidecar needs its own saved filename, we can't just use the same file for each.

I think it's worth getting this right here because this is going to be core to testing the pagination as I understand it.